sdk/function_id.go

```diff
@@ -0,0 +1,72 @@
+package sdk
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+
+	"github.com/capeprivacy/cli/entities"
+)
+
+const capeURL = "https://app.capeprivacy.com"
+
+type FunctionIDRequest struct {
+	UserName     string
+	FunctionName string
+	URL          string
+}
+
+type errorMsg struct {
+	Message string `json:"message"`
+}
+
+func GetFunctionID(functionReq FunctionIDRequest) (string, error) {
```

Comment:
I kinda think explicit arguments would make more sense for this function ... 

`sdk.GetFunctionID("shweta", "echo")` rather than this request object

Reply:
hmm, it would then be `sdk.GetFunctionID("shweta", "echo", "hardcoded_url") `
I kept the url included in the request object to make testing it easier. Like here, it passes the test server's url instead of `app.capeprivacy.com` to call the test server instead.https://github.com/capeprivacy/cli/blob/8035dcf17b390a06f49be304ee9ea0ba41ca4fe8/sdk/function_id_test.go#L34